acceptanceTests/cypress/integration/tests/Rabbit.js

```diff
@@ -1,8 +1,8 @@
 import {checkFilterByMethod, valueTabs,} from "../testHelpers/TrafficHelper";
 
-
 it('opening mizu', function () {
     cy.visit(Cypress.env('testUrl'));
+    cy.get('#total-entries').invoke('text').should('match', /^[4-7][0-9]$/m)
```

Comment:
same here, convert to number and do >= 40 and < 80 or something like this

Reply:
I need to use here the should(), If I convert the entries number to int I will need to use expect() instead, which is not good for this case.